### DISCUSSION TITLE
tooltips

### DISCUSSION THREAD

**Discussion by alexhoffenreich** [nasa/openmct#7551] [Community Q&A]
Is it possible to add custom tooltips to telemetry point?

**Comment by ozyx** [comment]
Hi @alexhoffenreich !

Possibly. I want to make sure I understand properly, though. What do you mean by tooltips? Could you provide an image?

Not sure if this is what you're looking for, but if you hold down the `ctrl` key and hover over certain elements such as tree items, plot legend items, object names, etc. it displays a tooltip with the path to the object:


https://github.com/nasa/openmct/assets/9259993/bddec1f9-c060-4b4a-b1da-434c2dbbbc00

**Reply by alexhoffenreich** [reply]
Exactly like that, but instead of the path, can it display something else, like the notes ?
I mean, the "notes" attribute of the telemetry point node in the dictionary json.

**Comment by jvigliotta** [comment]
@alexhoffenreich Currently for tree items we show the object path, as you saw above. This is setup here: https://github.com/nasa/openmct/blob/master/src/ui/components/ObjectLabel.vue#L164-L167

The only way to do the notes instead, would be to modify that code like so:

``` javascript
showToolTip() {
      const { BELOW } = this.openmct.tooltips.TOOLTIP_LOCATIONS;
      if (this.domainObject.notes) {
        this.buildToolTip(this.domainObject.notes, BELOW, 'objectLabel');
      }
    }
```

Which would give you:
![Screenshot 2025-06-11 at 4 01 13 PM](https://github.com/user-attachments/assets/9ae70fb5-d523-4440-912c-947412693a12)

We don't currently have a way to customize that, but if you're ok modifying the code, that's one way to do it. Tool times aren't very customizable at this point.

**Comment by akhenry** [comment]
@jvigliotta
